docs/lists_bank_holidays.md

```diff
@@ -0,0 +1,26 @@
+## Bank Holidays
+
+Bank Holidays in England and Wales (taken from GOV.UK)
+
+```ruby
+require 'dfe/reference_data/bank_holidays`
+```
+
+### `DfE::ReferenceData::BankHolidays::BANK_HOLIDAYS`
+
+Bank Holidays.
+
+Owner: None.
+
+Users: None.
+
+Source: https://www.gov.uk/what-different-qualification-levels-mean/list-of-qualification-levels
```

Comment:
Is this a Copy+Paste error ?This source seems to be related to qualifications.

Reply:
Good spot! Now updated!